### PR TITLE
Add a Cron4s config writer

### PR DESCRIPTION
**Pull request by alonsodomin**

### CODE REVIEW

File: modules/cron4s/src/main/scala/pureconfig/module/cron4s/package.scala [hunk 1]
```diff
@@ -2,14 +2,17 @@ package pureconfig.module
...
 package object cron4s {
 
-  implicit val cronExprReader: ConfigReader[CronExpr] =
+  implicit val cronExprConfigReader: ConfigReader[CronExpr] =
     ConfigReader.fromString(str =>
       Cron.parse(str).fold(
         err => Left(CannotConvert(str, "CronExpr", err.getMessage)),
         cronExpr => Right(cronExpr)))
+
+  implicit val cronExprConfigWriter: ConfigWriter[CronExpr] =
```
Comment (ruippeixotog) [inline modules/cron4s/src/main/scala/pureconfig/module/cron4s/package.scala]: Can you combine the reader and writer parts in a `ConfigConvert`? You should be able to do that easily with `ConfigConvert.viaString` (see [pureconfig-joda](https://github.com/pureconfig/pureconfig/blob/master/modules/joda/src/main/scala/pureconfig/module/joda/package.scala) for examples).

**Review by ruippeixotog — CHANGES_REQUESTED** [review]
Thanks for the contribution, @alonsodomin!

File: modules/cron4s/src/main/scala/pureconfig/module/cron4s/package.scala (hunk 1 above)
Comment (alonsodomin) [reply modules/cron4s/src/main/scala/pureconfig/module/cron4s/package.scala]: @ruippeixotog thanks to the pointer to `ConfigConvert`, wasn't aware of its existance. Should be sorted now.

**Review by jcazevedo — APPROVED** [review]
Looks good. Thanks for this, @alonsodomin!

**Review by ruippeixotog — APPROVED** [review]

**Comment by leifwickland** [comment]
@alonsodomin Thanks for your contribution. Please consider sending a PR to add yourself to the [pureconfig contributors list](https://github.com/pureconfig/pureconfig/blob/master/CONTRIBUTORS).

**Pull request merged**